base/src/commands.rs

```diff
@@ -14,6 +14,14 @@ pub trait TpmCommand: Marshalable {
     type RespHandles: Marshalable;
 }
 
+/// ============================================================================
```

Comment:
These block separator comments (`// =====...`) should probably not be doc comments (e.g. don't use 3 forward slashes). They're here for the developer's sake, not the sake of the types.

You can see what happens to these comments if you run `cargo doc`, in this case the block comments are applied to the command that they are written before.

![image](https://github.com/user-attachments/assets/4f4089ba-0f18-462c-bdc5-bffae23debfe)

This is very likely not what you intended.

Reply:
Thank you for the great input 👍 🙂 I agree, that's not what was intended.

I would probably go for removing the chapter blocks all together, and just have a single reference to TPM spec in each function request definition, which can still be used as a guide through the source code. I will try to restructure at upload again 🙂

Reply:
Thanks! Perfect, I think that works good.